Observer/OrderSaveAfter.php

```diff
@@ -30,15 +44,29 @@ function execute(Observer $ob) {
 				if ($cfg->getValue('stock2shop/order_export/enable', SS::SCOPE_STORE, $o->getStore())) {
 					/** @var string $state */ /** @var string $status */
 					list($state, $status) = [$o->getState(), $o->getStatus()];
-					try {$res = self::post(Payload::get($o), $o->getStore());} /** @var string $res */
```

Comment:
It looks like we've lost the just-in-case try-catch block here (for exceptions other than the ones that we are explicitly handling)...

I think we'd better put it back in

Reply:
I think if we enclose this in a try - catch:

```
					$payload = Payload::get($o);
					$encoded_str = self::encode($payload);
					$order_id = $o->getIncrementId();
					$payload_str = !empty(self::$encoding_error_msg)
						? '{"error": "Magento webhook failed to encode order, please look at order ' . $order_id . ' on website to see the details."}'
						: $encoded_str;
					$res = self::post($payload_str, $o->getStore());
```
And then if there is an exception:

1. Log it to the logger
2. Add it to the errors list that gets returned by getErrors

Reply:
This is done